stable/toxiproxy/templates/deployment.yaml

```diff
@@ -0,0 +1,115 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "toxiproxy.fullname" . }}
+  labels:
+{{ include "toxiproxy.labels" . | indent 4 }}
+{{- with .Values.deploymentAnnotations }}
+annotations:
+  {{- toYaml . | nindent 4 }}
+{{- end }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: {{ include "toxiproxy.name" . }}
+      app.kubernetes.io/instance: {{ .Release.Name }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: {{ include "toxiproxy.name" . }}
+        app.kubernetes.io/instance: {{ .Release.Name }}
```

Comment:
Getting a linting error here but not too sure what the problem is. 

```
[ERROR] templates/deployment.yaml: unable to parse YAML
	error converting YAML to JSON: yaml: line 21: mapping values are not allowed in this context

Error: 1 chart(s) linted, 1 chart(s) failed
```

Reply:
Classic informative Helm error. I think my comment above.

Reply:
Fixed the wrong spacing on `annotations`. Still getting the same error though.